content/influxdb3/core/admin/tokens/admin/list.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: List an admin token
+title: List admin tokens
 description: >
-  Use the `influxdb3 show tokens` command
-  to list the [admin token](/influxdb3/enterprise/admin/tokens/admin/) for your {{< product-name omit="Clustered" >}} instance.
-  An admin token grants access to all actions on the server.
+  Use the `influxdb3` CLI or the `/api/v3` HTTP API
+  to list admin tokens for your {{< product-name >}} instance.
+  Use the  `influxdb3 show tokens` command to list all tokens or use SQL to query token metadata directly from the `SYSTEM.TOKENS` table.
```

Comment:
We probably ought to establish a capitalization convention for system tables. The actual table name is `system.tokens` (lower case). If you were to enforce case by wrapping these in double-quotes `"SYSTEM"."TOKENS"`, you'd get a `table 'public.SYSTEM.TOKENS' not found` error.

I know case doesn't really matter in this particular case, but I'd suggest using the actual lower-case table names.

```suggestion
  Use the  `influxdb3 show tokens` command to list all tokens or use SQL to query token metadata directly from the `system.tokens` table.
```

Reply:
Thanks for fixing all these. I obv. didn't think about that.